docs/install_guides/installing-openvino-linux.md

```diff
@@ -320,16 +320,21 @@ cd /opt/intel/openvino/install_dependencies/
 ```sh
 sudo -E su
 ```
-3. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics:
+3. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. Run the installation script with super user rights:
 ```sh
-./install_NEO_OCL_driver.sh
-```
-You may see the following command line output:
-
+sudo -E ./install_NEO_OCL_driver.sh
```

Comment:
do we need `sudo -E` here if we entered superuser mode in the previous step?

Reply:
Actually not